omnigibson/action_primitives/starter_semantic_action_primitives.py

```diff
@@ -1784,62 +1729,75 @@ def _rotate_in_place(self, end_pose, angle_threshold=m.DEFAULT_ANGLE_THRESHOLD):
         empty_action = self._empty_action()
         yield self._postprocess_action(empty_action)
 
-    def _sample_pose_near_object(self, obj, pose_on_obj=None, **kwargs):
+    def _sample_pose_near_object(
+        self,
+        obj,
+        pose_on_obj=None,
```

Comment:
In general, I think the term `pose_on_obj` is quite confusing. Is it a pregrasp pose? Is it a grasp pose? By the end of the day, it's just an arbitrary eef pose to reach. In some way, you should either pass in obj or `pose_on_obj`, but not both, right?